### ISSUE TITLE
Creating a flowwie Preference Pack

### ISSUE THREAD

**Issue by luzpaz** [Flowwie/flowwie-freecad#13]
Create a custom pack for users to download and enable. Here's an example of @Chrismettal's https://github.com/Chrismettal/FreeCAD-Behave-Dark-Preference-Pack 
See https://github.com/FreeCAD/FreeCAD-addons/pull/207

**Comment by Flowwie** [comment]
good idea, I will have a look. At the moment I am not 100% satisfied with my preferences for the FreeCAD 0.20, will have to make some smaller adjustments. Thanks!
